docs/core/install/includes/verify-download-macos-linux.md

```diff
@@ -0,0 +1,61 @@
+---
+author: adegeo
+ms.author: adegeo
+ms.date: 06/21/2023
+ms.topic: include
+---
+
+Use the `sha512sum` command to print the checksum of the file you've downloaded. For example, the following command reports the checksum of the _dotnet-sdk-7.0.304-linux-x64.tar.gz_ file:
+
+```bash
+$ sha512sum dotnet-sdk-7.0.304-linux-x64.tar.gz
+f4b7d0cde432bd37f445363b3937ad483e5006794886941e43124de051475925b3cd11313b73d2cae481ee9b8f131394df0873451f6088ffdbe73f150b1ed727  dotnet-sdk-7.0.304-linux-x64.tar.gz
+```
+
+Compare the checksum with the value provided by the download site.
+
+> [!IMPORTANT]
+> Even though a Linux file is shown in these examples, this information equally applies to macOS.
+
+### Use a checksum file to validate
+
+The .NET release notes contain a link to a checksum file you can use to validate your downloaded file. The following steps describe how to download the checksum file and validate a .NET install binary:
+
+01. The release notes page for .NET 7.0 on GitHub at <https://github.com/dotnet/core/tree/main/release-notes/7.0> contains a section named **Releases**. The table in this section links to the downloads and checksum files for each .NET 7 release:
+
+    :::image type="content" source="../media/install-sdk/release-notes-root.png" alt-text="The github release notes version table for .NET":::
+
+01. Select the link to your version of .NET that you downloaded. The previous section used .NET SDK 7.0.304, which is in the .NET 7.0.7 release.
+
+    > [!TIP]
+    > If you're not sure which .NET release contains your checksum file, explore the links until you find it.
```

Comment:
Not sure this is helpful?

Reply:
Yeah, probably not. There just isn't a one-to-one mapping and I just have a feeling people will arrive there and go "Which one do I download?!"

Reply:
The answer is clear in the release notes though, interestingly, I am unable to find links to release notes for a given release. Seems like a release notes link on the download pages (e.g. https://dotnet.microsoft.com/en-us/download/dotnet/7.0) would be useful? 

@mairaw - I don't recall if we ever had release notes linked from the download pages? URLs are in releases.json files.

Reply:
You could direct them to the release notes, which makes it fairly clear.